praw/models/reddit/live.py

```diff
@@ -191,21 +191,21 @@ def update(
 
         .. code-block:: python
 
-           thread = reddit.live('ukaeu1ik4sw5')
-           thread.contributor.update('spez')
+           thread = reddit.live("ukaeu1ik4sw5")
+           thread.contributor.update("spez")
 
-        To grant 'access' and 'edit' permissions (and to remove other
+        To grant "access" and "edit" permissions (and to remove other
         permissions), try:
```

Comment:
This should be double backticked.
```suggestion
        To grant ``"access"`` and ``"edit"`` permissions (and to
        remove other permissions), try:
```